packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -199,7 +198,7 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
 
         this.menuElement = newMenuElement;
 
-        this.interactionManager.pause(INTERACTION_ID);
+        this.interactionManager.pause(TOOLTIP_ID);
```

Comment:
I guess should probably rename this constant?

Reply:
Fair - added `PAUSE_TYPE`.